src/bijectors/leaky_relu.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    LeakyReLU{T, N}(α::T) <: Bijector{N}
+
+Defines the invertible mapping
+
+    x ↦ x if x ≥ 0 else αx
+
+where α > 0.
+"""
+struct LeakyReLU{T, N} <: Bijector{N}
+    α::T
+end
+
+LeakyReLU(α::T; dim::Val{N} = Val(0)) where {T<:Real, N} = LeakyReLU{T, N}(α)
+LeakyReLU(α::T; dim::Val{N} = Val(D)) where {D, T<:AbstractArray{<:Real, D}, N} = LeakyReLU{T, N}(α)
+
+up1(b::LeakyReLU{T, N}) where {T, N} = LeakyReLU{T, N + 1}(b.α)
+
+# (N=0) Univariate case
+function (b::LeakyReLU{<:Any, 0})(x::Real)
+    mask = x < zero(x)
+    return mask * b.α * x + !mask * x
+end
+(b::LeakyReLU{<:Any, 0})(x::AbstractVector{<:Real}) = map(b, x)
+
+Base.inv(b::LeakyReLU) = LeakyReLU(inv.(b.α))
```

Comment:
The dimension can't be inferred:
```suggestion
function Base.inv(b::LeakyReLU{<:Any,N}) where N
    invα = inv.(b.α)
    return LeakyReLU{typeof(invα),N}(invα)
end

```